data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -40,12 +40,12 @@
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "After a year and a day of grim battle, the city fell. The streets of  Dwarven Doors ran with gore. The dwarves managed to retreat into the caverns Knalga, but the humans who survived the sack were enslaved. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their subjugation could not be averted forever."
+            story= _ "After a year and a day of grim battle, the city fell. The streets of Dwarven Doors ran with gore. The dwarves managed to retreat into the caverns Knalga, but the humans who survived the sack were enslaved. Cut off from contact with the surface, the dwarves of Knalga fought on, but knew their subjugation could not be averted forever."
             {NR_BIGMAP}
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "Dark years passed; human generations grew up in slavery, and came near to forgetting that their ancestors had been free — and might have completely forgotten were it not for a fateful day in 518YW. The day that changed the lives of the people of Dwarven Doors — forever."
+            story= _ "Dark years passed; human generations grew up and died in slavery, and with every passing year they and came close to forgetting that their ancestors had ever been free. They might have completely forgotten were it not for a fateful day. The day that changed the lives of the people of Dwarven Doors — forever."
```

Comment:
Personally I'd put a comma after "passing year", but that's just my style. Also, "they and" sounds weird?